dragonfly/server.go

```diff
@@ -187,6 +187,26 @@ func (server *Server) Player(uuid uuid.UUID) (*player.Player, bool) {
 	return nil, false
 }
 
+// GetPlayer returns a player is the name passed matches part of a name of an online player: (in-case sensitive)
+func (server *Server) GetPlayer(name string) (*player.Player, bool) {
```

Comment:
This is pointless in my opinion, only use for it is searching for usernames gotten from players running commands, but won't be needed since a "target" argument will most likely be added.